components/list/list-item-generic-layout.js

```diff
@@ -530,6 +530,7 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 		if (!this.gridActive) return;
 		const slot = (e.path || e.composedPath()).find(node =>
 			node.nodeName === 'SLOT' && node.classList.contains('d2l-cell'));
+		if (!slot) return;
```

Comment:
The following line results in a JavaScript error in nested lists when focusing on `d2l-pager-load-more` because the `slot` would be undefined in that case.